share/completions/solc.fish

```diff
@@ -0,0 +1,83 @@
+# General Information
+complete -c solc -l help -d "Show help message and exit."
+complete -c solc -l version -d "Show version and exit."
+complete -c solc -l license -d "Show licensing information and exit."
+complete -c solc -l input-file -d "input file"
```

Comment:
this probably wants a file argument?

```suggestion
complete -c solc -l input-file -d "input file" -r
```

same for base-path, include-path, (maybe even allow-paths), output-dir, etc.